Kinvey/Kinvey/Result.swift

```diff
@@ -8,9 +8,13 @@
 
 import Foundation
 
+/// Enumeration that represents a result expected usually after an async call
```

Comment:
Is the `expected usually after async call` necessary? How about just `represents the result of an operation / API`